Content.Server/Dragon/Components/DragonComponent.cs

```diff
@@ -32,6 +32,9 @@ public sealed partial class DragonComponent : Component
         [ViewVariables(VVAccess.ReadWrite), DataField("riftAccumulator")]
         public float RiftAccumulator = 0f;
 
+        [ViewVariables(VVAccess.ReadWrite), DataField("riftAccumulatorActive")]
+        public bool RiftAccumulatorActive = true;
+
```

Comment:
Please update this to use modern standards as VVRW is unnecessary and datafield strings are obsolete though realistically this needs removing.